source/features/set-default-repositories-type-to-sources.tsx

```diff
@@ -10,9 +10,11 @@ function addSourceTypeToLink(link: HTMLAnchorElement): void {
 // No `include`, no `signal` necessary
 function init(): void {
 	observe([
-		'.header-nav-current-user ~ a[href$="tab=repositories"]', // "Your repositories" in the header profile dropdown
-		'[aria-label="User profile"] a[href$="tab=repositories"]', // "Repositories" tab on user profile
+		`a:is([href^="/"], [href^="${location.origin}"])[href$="?tab=repositories"]`, // "Repositories" tab on user profile
```

Comment:
This is the only one needed

Reply:
Other links doesn't match `?tab=repositories` though?

Reply:
Nice selector, it would be good to explain the `is` part

Reply:
Right, there should be another generic one then, or else this link won't be covered either I think:

- https://github.com/orgs/refined-github/repositories

```
https://github.com/orgs/refined-github/repositories
```

Reply:
By the way CSS Nesting is coming soon 😍 and it would simplify a lot of our selectors: https://caniuse.com/css-nesting

- Chrome has it
- Safari 16.5 has it (maybe June)
- Firefox eh https://bugzilla.mozilla.org/show_bug.cgi?id=1648037

Reply:
This wasn't addressed. We should not depend on any classes.


> Now it still misses direct links to organization profiles, i.e.
> 
> ```
> https://github.com/refined-github
> ```

This feature does not target those links, it should target every:

- `/*?tab=repositories`
- `/orgs/*/repositories/` 

otherwise we're back to square one (i.e. "some links don't work")

Also I'd change the name of the feature to something more reasonable because such long features are impossible to mention in commit titles. I'd go for `hide-user-forks` maybe